docs/tempo/website/setup/linux.md

```diff
@@ -0,0 +1,299 @@
+   ```
+
+1. Download the enterprise-traces binary, verify checksums, and add network capabilities to the binary:
+
+   ```bash
+   curl -Lo /usr/local/bin/enterprise-traces \
+   https://dl.grafana.com/get/releases/enterprise-traces-v1.3.0-linux-amd64
+   echo d950922d2038c84620ebe63a21786b9eaf8d4ed5f1801e6664133520407e5e86 \
+     /usr/local/bin/enterprise-traces | sha256sum -c
+   chmod 0755 /usr/local/bin/enterprise-traces
+   setcap 'cap_net_bind_service=+ep' /usr/local/bin/enterprise-traces
+   ```
+
+1. Set up systemd unit and enable startup on boot:
+
+   ```bash
+   cat > /etc/systemd/system/enterprise-traces.service <<EOF
+   [Unit]
+   After=network.target
+
+   [Service]
+   User=enterprise-traces
+   Group=enterprise-traces
+   WorkingDirectory=/var/lib/enterprise-traces
+   ExecStart=/usr/local/bin/enterprise-traces \
+   -config.file=/etc/enterprise-traces/enterprise-traces.yaml \
+   -log.level=warn \
+
+   [Install]
+   WantedBy=default.target
+   EOF
+
+   systemctl daemon-reload
+   systemctl enable enterprise-traces.service
+   ```
+
+## Create a Tempo configuration file
+
+Copy the following YAML configuration to a file called `enterprise-traces.yaml`.
+
+Paste in your S3 credentials for admin_client and the storage backend. If you wish to give your cluster a unique name, add a cluster property with the appropriate name. If you do not add a cluster name this will be taken automatically from the license.
+By default, the `cluster_name` Update the `cluster_name` field with the name of the cluster your license was issued for and paste in your S3 credentials for the `admin_client`.
+
+Refer to the [Tempo configuration documentation]({<< relref "../../configuration" >>}) for explanations of the available options.
+
+In the following configuration, Tempo options are altered to only listen to the OTLP gRPC and HTTP protocols.
+By default, Tempo listens for all compatible protocols.
+The extended instructions for installing the TNS application and Grafana Agent to verify that Tempo is receiving traces relies on the default Jaeger port being available, hence disabling listening on that port in Tempo for a single Linux node.
+
+```yaml
+multitenancy_enabled: true
+auth:
+  type: enterprise
+
+license:
+  path: /etc/enterprise-traces/license.jwt
+
+http_api_prefix: /tempo
+
+distributor:
+  receivers:
+    otlp:
+      protocols:
+        grpc:
+        http:
+
+ingester:
+    lifecycler:
+      ring:
+        replication_factor: 3
+
+server:
+    http_listen_port: 3200
+
+storage:
+    trace:
+        backend: s3
+        s3:
+          endpoint: s3.us-east-1.amazonaws.com
+          bucket: grafana-traces-data
+          forcepathstyle: true
+          #set to true if endpoint is https
+          insecure: true
+          access_key: # TODO: insert your key id
+          secret_key: # TODO: insert your secret key
+        wal:
+          path: /var/lib/enterprise-traces/wal
+
+admin_api:
```

Comment:
`admin_api` and `admin_client` below can get dropped for Tempo.

Reply:
Done.